learn/generation/rag-for-hybrid/rag-for-hybrid-search.ipynb

```diff
@@ -0,0 +1,2569 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    query_sembedding = bm25.encode_documents(query)
</pre><p> </p><p>bm25.encode_query</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/284/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='96'/>

Reply:
<p>Oh, shoot, thank you.</p>